### DISCUSSION TITLE
【项目组队】有看过 日本动漫“赌博默示录” 或者 李易峰演的“动物世界” 的朋友吗？

### DISCUSSION THREAD

**Discussion by zedz** [IntensiveCoLearning/Web3-URL#147] [Ideas and Project Teams]
感兴趣的朋友可以看看b站有赌博默示录动漫全集，应该也有动物世界的解说，十几分钟就可以了解。
看完应该会对里面的游戏有比较直观的理解。如果对这种刺激的石头剪刀布也感兴趣，可以查看我们的：

Notion文档
[Web3 E-Card](https://www.notion.so/Web3-E-Card-6bcf5fc4ac2b4e36a801deb66af5c937?pvs=4)

墨刀原型
[原型初稿](https://modao.cc/proto/jWr1I6Rsh2n78XPgtXl1w/sharing?view_mode=read_only) #web3Ecard-分享

Github仓库
[Outlaw Voyage](https://github.com/OutlawVoyage/OutlawVoyage)
目前正式命名游戏为Outlaw Voyage

**Comment by jjeejj** [comment]
申请组队

**Reply by qizhou** [reply]
看了几集《赌博默示录》，太有意思了

**Reply by zedz** [reply]
非常非常好看！！是过去很久都非常记得的作品！！

**Reply by qizhou** [reply]
看了几集，已经看到：
- 囚徒困境
- 联合坐庄
- 抢庄/黑吃黑
- 预判敌人的预判

堪称web3入门教材😄

**Reply by zedz** [reply]
😄 嘿嘿  很开心推荐的能被喜欢！

**Comment by qizhou** [comment]
> 感兴趣的朋友可以看看b站有赌博默示录动漫全集，应该也有动物世界的解说，十几分钟就可以了解。 看完应该会对里面的游戏有比较直观的理解。如果对这种刺激的石头剪刀布也感兴趣，可以查看我与punkcan对话的整理文档：
> 
> [Web3 E-Card](https://www.notion.so/Web3-E-Card-6bcf5fc4ac2b4e36a801deb66af5c937?pvs=4)

链接无法打开，需要分享权限。

**Reply by zedz** [reply]
权限都通过了

不想申请权限的朋友可以试试这个链接：
[web3 E-card online](https://gigantic-bandicoot-5c1.notion.site/Web3-E-Card-6bcf5fc4ac2b4e36a801deb66af5c937)

**Comment by EthanShang8989** [comment]
有群链接嘛

**Reply by zedz** [reply]
辛苦加我wx，huanyuhein
